packages/web-app-files/src/mixins/actions/copy.js

```diff
@@ -53,9 +56,19 @@ export default {
   },
   methods: {
     ...mapActions('Files', ['copySelectedFiles']),
-
-    $_copy_trigger() {
+    ...mapMutations('Files', ['SET_FILE_SELECTION']),
+    $_copy_trigger({ resources }) {
+      this.SET_FILE_SELECTION(resources)
       this.copySelectedFiles({ space: this.space })
+      const currentFolderSelected = resources.some((r) => r.id === this.currentFolder?.id)
+      if (currentFolderSelected) {
+        return this.$router.push(
+          createFileRouteOptions(this.space, {
```

Comment:
Are you navigating to the parent because otherwise the selection doesn't make sense? Or what's the reason?

Reply:
Yes. No matter if you copy or cut, you can't perform the action within the current folder. I admit, it might be a bit confusing for the user though. Do you have an alternative solution?

Reply:
Accessibility wise (and probably also in general UX wise) it's an issue to navigate to another page without communicating clearly. As far as I can tell we need to stay in the current folder. I'd be fine with the notification.